packages/devtools_app/lib/src/screens/inspector_v2/widget_properties/properties_view.dart

```diff
@@ -96,21 +97,44 @@ class PropertiesTable extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final sortedProperties = _sortPropertiesByLevel(properties);
     return Scrollbar(
       controller: scrollController,
       thumbVisibility: true,
       child: ListView.builder(
         controller: scrollController,
-        itemCount: properties.length,
+        itemCount: sortedProperties.length,
         itemBuilder: (context, index) {
           return PropertyItem(
             index: index,
-            properties: properties,
+            properties: sortedProperties,
           );
         },
       ),
     );
   }
+
+  /// Filters out properties with [DiagnosticLevel.hidden] and sorts properties
+  /// with [DiagnosticLevel.fine] behind all others.
+  List<RemoteDiagnosticsNode> _sortPropertiesByLevel(
```

Comment:
nit: rename to `_sortAndFilterPropertiesByLevel` since this method also filters

Reply:
Done!